### PR TITLE
EPMRPP-50944: [AGENT]. Jasmine Finish launch/suite/test/ step with populated status

### PR DESCRIPTION
**Pull request by KatsiarynaTatarynovich**
Tests will pass after merge of this PR https://github.com/reportportal/client-javascript/pull/73

### CODE REVIEW

File: lib/jasmine-reportportal-reporter.js [hunk 1]
```diff
@@ -23,6 +24,7 @@ class ReportportalReporter {
         this.suiteLogs = new Map();
         this.suiteAttributes = new Map();
         this.suiteTestCaseIds = new Map();
+        this.suiteStatus = new Map();
```
Comment (ElenaRomanchuk) [inline lib/jasmine-reportportal-reporter.js:27]: May be `suiteStatuses`?

File: lib/jasmine-reportportal-reporter.js [hunk 2]
```diff
@@ -101,6 +103,18 @@ class ReportportalReporter {
...
+    setStatus(data) {
+        if (data && data.suite) {
+            this.suiteStatus.set(data.suite, data.status);
+        } else {
+            this.additionalCustomParams = Object.assign(this.additionalCustomParams, data && { customStatus: data.status });
+        }
+    }
+
+    setLaunchStatus(status) {
+        this.conf.launchStatus = status;
```
Comment (ElenaRomanchuk) [inline lib/jasmine-reportportal-reporter.js:115]: I'm not sure that it's good idea to modify config in runtime.

File: README.md [hunk 3]
```diff
@@ -138,6 +138,35 @@ PublicReportingAPI provides the corresponding methods for reporting logs into th
...
+* setLaunchStatus(*status*). Assign *status* to the launch.<br/>
+*status* should be equal to one of the following values: *passed*, *failed*, *stopped*, *skipped*, *interrupted*, *cancelled*, *info*, *warn*.<br/>
+
+* setLaunchStatusPassed(). Assign *passed* status to the launch.
+* setLaunchStatusFailed(). Assign *failed* status to the launch.
+* setLaunchStatusSkipped(). Assign *skipped* status to the launch.
+* setLaunchStatusStopped(). Assign *stopped* status to the launch.
+* setLaunchStatusInterrupted(). Assign *interrupted* status to the launch.
+* setLaunchStatusCancelled(). Assign *cancelled* status to the launch.
+* setLaunchStatusInfo(). Assign *info* status to the launch.
```
Comment (AmsterGet) [inline README.md:167]: Does the launch supports `info` and `warn` statuses?

Comment (KatsiarynaTatarynovich) [reply README.md:167]: According to the answer from the backend of the developer, the statuses for the suite and step are also suitable for the launch

File: lib/jasmine-reportportal-reporter.js (hunk 2 above)
Comment (KatsiarynaTatarynovich) [reply lib/jasmine-reportportal-reporter.js:115]: Unfortunately, the other way with the callback did not work.

File: README.md (hunk 3 above)
Comment (AmsterGet) [reply README.md:167]: Got it, thank you

**Pull request merged**